pages/dashboard/DashboardHandler.inc.php

```diff
@@ -112,6 +112,33 @@ function index($args, $request) {
 				]
 			);
 			$lists[$activeListPanel->id] = $activeListPanel->getConfig();
+			$lists[$activeListPanel->id]['filters'][] = [
+				'filters' => [
+					[
+						'title' => _('editors'),
+						'param' => 'assignedTo',
+						'value' => [],
+						'filterType' => 'pkp-filter-autosuggest',
+						'autosuggestProps' => [
+								'allErrors' => (object) [],
+								'apiUrl' => $request->getDispatcher()->url($request, ROUTE_API, $context->getPath(), 'users', null, null, ['roleIds' => [ROLE_ID_MANAGER, ROLE_ID_EDITOR]]),
```

Comment:
The roles here should be `[ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR]`.

You can find what role constants are defined by searching for `defined('ROLE_ID`.